Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -66,14 +66,26 @@ def from_string(self, string):
         return None
 
     def credentials(self, required=True):
-        return credentials(
+        args = dict(
             url=self.api_url,
             required=required,
             name=self.url.split('/')[2].replace('.', '_').upper(),
-            prompt="GitHub's API\nPlease generate a 'Personal access token' via 'Developer settings' for your user",
+            prompt='''GitHub's API
```

Comment:
Nit: might consider re-wording something like:
Please go to https://{host}/settings/tokens and generate a new Personal access token. Please select 'repo' and 'workflow' scopes, and appropriate 'Expiration' duration.